examples/SN99938_meta.json

```diff
@@ -0,0 +1,46 @@
+{
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [
+            31.4643,
+            80.1058
+        ]
+    },
+    "properties": {
+        "pubtime": "2019-09-03T10:01:35.325086+00:00",
+        "title": "Observations from station KVIT\u00d8YA SN99938",
+        "data_id": "metno_obs_SN99938",
+        "start_datetime": "1996-01-01T03:00:00.000000000",
+        "end_datetime": "2019-09-03T10:00:00.000000000",
```

Comment:
I e-soh skal vi kun håndtere sanntidsdata, så for vår del vil tidspunktet være nær nåtid og dekke et kortere tidsintervall. Jeg er litt usikker på størrelsen på datapakkene, men jeg lurer på om vi ble enig om å sende hver enkelt observasjon.. Evt kan det kortes ned til 5-10 minutters intervaller? Husk at dataene også skal ligge i meldingen, så det må uansett ikke bli for mye data.